bionic/flow.py

```diff
@@ -843,11 +854,40 @@ def entity_protocol(self, name):
         ----------
 
         name: String
-            The name of a entity.
+            The name of an entity.
         """
 
         return self._state.get_provider(name).protocol_for_name(name)
 
+    def entity_docstring(self, name):
+        """
+        Returns the docstring for entity ``name`` if a docstring is defined,
```

Comment:
nit: "for entity `name`" reads a little weird to me.  WDYT about "for the named entity"?